jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibExtension.java

```diff
@@ -143,4 +145,21 @@ boolean getAllowInsecureRegistries() {
     }
     return allowInsecureRegistries.get();
   }
+
+  @Input
+  @Nullable
+  @Optional
+  // TODO: validate the value somewhere. We don't have a central place for config validation yet.
+  // Or, can this be enforced at the plugin level, e.g., by having this as an enum in both Maven and
+  // Gradle?
+  public String getPackagingOverride() {
```

Comment:
should we allow packagingOverride="jar" for "war" projects?

Reply:
Currently, any value other than "war" will be considered the "flat" packaging (no error). I hope an invalid value will return an error when validation is set in place. So, yes, packagingOverride="jar" will force the "flat" packaging on a project with `<packaging>war` in `pom.xml`, for example.